ci/resources.yml

```diff
@@ -2,16 +2,17 @@ resources:
 - name: orchestration-service-pr
   type: pull-request
   source:
-    access_token: ((github_com.token))
+    access_token: ((dataworks-secrets.enterprise_github_pat))
     repo: dwp/orchestration-service
     uri: https://github.com/dwp/orchestration-service.git
-  check_every: 15m
+  check_every: 5m
+  webhook_token: ((dataworks.concourse_github_webhook_token))
+
 - name: orchestration-service
   type: git
   source:
     branch: master
-    username: ((github_com.username))
-    password: ((github_com.token))
+    access_token: ((dataworks-secrets.enterprise_github_pat))
     uri: https://github.com/dwp/orchestration-service.git
   check_every: 15m
```

Comment:
you should set the check to `720h` and add the webhook in `dataworks-github-config`